metadata/src/main/java/org/apache/kafka/metadata/properties/MetaProperties.java

```diff
@@ -47,7 +47,7 @@ public final class MetaProperties {
     /**
      * The property that specifies the node id. Replaces broker.id in V1.
      */
-    static final String NODE_ID_PROP = "node.id";
+    public static final String NODE_ID_PROP = "node.id";
```

Comment:
This is really not nice that we have to do that. I wonder if we need to wait for the `NodeIdProp` from KafkaConfig to move instead of doing that.

Reply:
Ok, agreed.